src/GravityCalculator.cpp

```diff
@@ -0,0 +1,18 @@
+#include "GravityCalculator.h"
+
+#include "utils/ArrayUtils.h"
+
+void GravityCalculator::calculateForces(ParticleContainer &particles) {
```

Comment:
Can we abstract this further?

If multiple forces are involved it may be usefull to store a List of all ForceFunctions somewhere, so that the Particle container can apply them to all pairs.

For this to work forces need to be additive, and cant just set the new f value